modules/e2ee/Worker.js

```diff
@@ -418,6 +445,19 @@ onmessage = async event => {
         } else {
             context.setKey(false, keyIndex);
         }
+    } else if (operation === 'ratchet') {
+        const { participantId } = event.data;
+
+        // TODO: can we ensure this is for our own sender key?
```

Comment:
We could bring back the initialize method with the local participant ID.

Reply:
long term we'll do the sender/receiver context so its ok to leave as a TODO